GPy/kern/src/stationary.py

```diff
@@ -222,6 +221,10 @@ def update_gradients_direct(self, dL_dVar, dL_dLen):
         """
         self.variance.gradient = dL_dVar
         self.lengthscale.gradient = dL_dLen
+    
+    def append_gradients_direct(self, dL_dVar, dL_dLen):
```

Comment:
Same here, could we do this directly where it is actually happening, instead of having these helpers?